File: KuantumCS/VeriPaketi.cs
using System;

public class VeriPaketi : KuantumNesnesi
{
    public VeriPaketi(string id, int stabilite, int tehlike)
        : base(id, stabilite, tehlike)
    {
    }

    public override void AnalizEt()
    {
        Stabilite -= 5;
        Console.WriteLine("Veri içeriği okundu.");
    }
}

File: KuantumPYTHON/veri_paketi.py
from kuantum_nesnesi import KuantumNesnesi

class VeriPaketi(KuantumNesnesi):
    def __init__(self, ID, stabilite, tehlike_seviyesi):
        super().__init__(ID, stabilite, tehlike_seviyesi)

    def analiz_et(self):
        # stabilite 5 düşer
        self.stabilite -= 5
        print("Veri içeriği okundu.")
